generator/src/pkg/service/util.go

```diff
@@ -301,7 +293,7 @@ func CreateInputCalledSvc() model.CalledService {
 	var calledSvc model.CalledService
 
 	calledSvc.Port = DefaultExtPort
-	calledSvc.Protocol = defaultProtocol
```

Comment:
isn't it better to keep the default protocol configurable as a parameter as before?

Reply:
Agreed, I don't know why I removed it. Fixed.